### ISSUE TITLE
[TD-31] gnutls-certtool vs certtool

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskserver#33]
_Axel Gschaider on 2013-10-10T05:38:38Z says:_ 

 The scripts to create a certificate all try to call (at)gnutls-certtool@ but here on my Debian machine ( to be precise: Raspbian ) the fitting binary in the package (at)gnutls-bin@ is called (at)certtool@

This was easily solved by editing all the generate.* files in /pki but I guess either this scripts should be aware of both binary names or the install instruction should state what package to use. Preferably the first :)

P.S.: I just confirmed this for Ubuntu too

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-10-10T05:38:38Z
Modified: 2014-02-09T02:19:49Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-10-10T11:01:01Z says:_ 

 This was fixed in commit 91187538c17680236a97641b08f398356b3b9d7a, three weeks ago.
